omero/sysadmins/python3-migration.rst

```diff
@@ -20,20 +20,14 @@ Basic steps
 Choosing a platform
 -------------------
 
-The two recommended platforms, CentOS 7 and Ubuntu 18.04, have Python 3.6 as
-default installation and have therefore received the most testing which is why
-Python 3.6 is the preferred version of Python.
+The two recommended platforms, Rocky Linux/RHEL 9 and Ubuntu 22.04,
+have Python 3.9 and 3.10 respectively as
+default installation.
 
-Both Python 3.5 and 3.7 should work and are slated to have support added, but
-Python 3.6 has been the focus of testing during the migration.
+All Python 3.6+ versions should work.
```

Comment:
With recent OMERO.py versions Python 3.8 should be the current minimal requirement (soon to be bumped to Python 3.9 as 3.8 is becoming EOL in the Fall)